app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -105,19 +146,60 @@ export default {
       return validator
     },
     selfBond() {
-      parseFloat(this.validator.tokens) -
-        parseFloat(this.validator.delegator_shares)
+      return this.validator.selfBond
+        ? (this.validator.selfBond * 100).toFixed(2)
+        : 0
+    },
+    myBond() {
+      return this.delegation.committedDelegates[this.validator.owner] || 0
+    },
+    powerRatio() {
+      return ratToBigNumber(this.validator.tokens)
+        .div(this.delegates.globalPower)
+        .toNumber()
+    },
+    powerRatioLevel() {
+      if (this.powerRatio < 0.01) return "green"
+      if (this.powerRatio < 0.03) return "yellow"
+      return "red"
+    },
+    commissionLevel() {
+      if (this.validator.commission < 0.01) return "green"
+      if (this.validator.commission < 0.03) return "yellow"
+      return "red"
+    },
+    status() {
+      // status: jailed
+      if (this.validator.revoked)
+        return "This validator has been jailed and is not currently validating"
+
+      // status: candidate
+      if (parseFloat(this.validator.voting_power) === 0)
+        return "This validator has declared candidacy but does not have enough voting power yet"
+
+      // status: validator
+      return "This validator is actively validating"
+    },
+    statusColor() {
+      // status: jailed
+      if (this.validator.revoked) return "red"
```

Comment:
can we change the background color of the validator header as well (cc: @jbibla )? I spent a couple minutes trying to figure out where I could check the validator status

Reply:
or border line color

Reply:
i think we should go ahead with this for now - but i am very happy to play around with more ways of making status more obvious. good feedback.

Reply:
Maybe have a color label?

Can one of you please create an issue for this?

Reply:
I actually think the problem relies in that we haven't yet implemented the same dots on the validators page. When I saw the original designs it was more clear that they were jailed. I'll make an issue for this

Reply:
See #1325